modules/ROOT/pages/configuring-properties.adoc

```diff
@@ -229,8 +229,9 @@ Configuration properties can be overwritten. The Hierarchy in which these are tr
 . System Properties
 . Environment Properties
 . Deployment Properties
-. Application Properties
+. Application Properties (includes Configuration Properties, Secure Configuration Properties and other Custom Configuration Properties)
```

Comment:
I suggest this rewrite for the Hierarchies section:

Configuration properties can be overwritten. The system evaluates these properties in the following order:
. System properties
. Environment properties
. Deployment properties
. Application properties
. Global properties
So, for example, if a configuration property named size is defined as both a system property and an application property, the value of the application property named size will override that of the system property named size.
Also, a global property can depend on application, environment, system and/or deployment properties. An application property can depend on deployment, environment, and/or system properties, and so on.

Can you explain what you mean by "depend on"?